juju-sync-watch

```diff
@@ -39,8 +42,9 @@ def parse_args():
     parser.add_argument('unit')
     parser.add_argument('charm_path', nargs='?')
     parser.add_argument('-d', '--description', action=ShowDesc, help='show description')
-    parser.add_argument('-e', '--environment', dest='env', default=JUJU_ENV,
-                        help='juju environment to operate in')
+    parser.add_argument('-e', '--environment', dest='env', default=current_environment(),
```

Comment:
That seems unnecessary.  -e is fine, i was wondering if we should normalize based on whats in juju2, but thats a fair point that this rides both juju 1.25 and juju 2. 👍